portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMCache.testcase

```diff
@@ -122,4 +123,35 @@ definition {
 			uncacheableMimeType = "true");
 	}
 
```

Comment:
@Jeremy-Ch Try to commit your code cleaner next time. You staged some changes which should have belonged in another commit (https://github.com/austinchiang/liferay-portal/pull/1287/commits/4b0e079a5971d7eb61d9781262794739d9895a18)